jenkinsfile

```diff
@@ -35,7 +35,7 @@ pipeline {
                         sh "docker rm psama || true"
                         sh """docker run --name=psama --restart always \
                             --network=picsure \
-                            --env-file /usr/local/docker-config/psama/.env \
+                            --env-file /usr/local/docker-config/psama/psama.env \
```

Comment:
This didn't work in staging but perhaps I could also just leave this alone and make sure it uses `.env` instead of `psama.env`.